clang/docs/CommandGuide/clang.rst

```diff
@@ -733,16 +733,17 @@ ENVIRONMENT
 
 .. envvar:: CPATH
 
-  If this environment variable is present, it is treated as a delimited list of
-  paths to be added to the default system include path list. The delimiter is
-  the platform dependent delimiter, as used in the PATH environment variable.
-
-  Empty components in the environment variable are ignored.
+  This environment variable specifies additional header file search paths which
+  behave as if they were specified with the :option:`-I\<directory\>` option at
+  the end of the driver command line. Paths are delimited by the platform
```

Comment:
There is a little bit of a "please figure/intuit out what this means" with the 'end of the driver command line' bit.  Could you clarify what the implications of that are?

Reply:
Hmm, I don't have any particularly good ideas that wouldn't duplicate what is (or should be) in the referenced description of the `-I` option. What I have now is pretty similar to what [gcc documents](https://gcc.gnu.org/onlinedocs/gcc/Environment-Variables.html#index-CPATH).

Reply:
I don't mind a quick 'high level' view of what that means, even if it means copying slightly from the -I option.  As far as the GCC docs, those aren't particularly good either, so we can aim for better than that.

Reply:
Thanks, @erichkeane, I changed the doc to instead state where these paths are added relative to other paths. I also made it more explicit that paths added via `CPATH` are non-system header search paths.